From efad24bb00ac2cd0a8a3c6a1df1cfd95307daa29 Mon Sep 17 00:00:00 2001
From: Andreas Metzler <ametzler@downhill.at.eu.org>
Date: Sun, 13 Dec 2009 10:41:42 +0100
Subject: [PATCH 1/3] Make docbook2texi executable name configurable.

Some systems, e.g. Debian, install the docbook2x variant of the
docbook2texi command as docbook2x-texi. Make the executable name
configurable in the Makefile.
--- a/doc-docbook/Makefile
+++ b/doc-docbook/Makefile
@@ -2,6 +2,9 @@
 # Copyright (c) The Exim Maintainers 2020
 export LC_ALL=C
 
+# Some systems install the docbook2x variant as docbook2x-texi.
+DOCBOOK2TEXI?=docbook2texi
+
 notarget:;    @echo "** You must specify a target, in the form x.y, where x is 'filter', 'spec',"
 	      @echo "** or 'test', and y is 'xml', 'fo', 'ps', 'pdf', 'html', 'txt', or 'info'."
 	      @echo "** One other possible targets 'exim.8', 'spec.utf8'".
@@ -107,7 +110,7 @@ filter.txt:   filter-txt.xml Tidytxt MyS
 # work.
 
 filter.info:  filter-info.xml
-	      docbook2texi filter-info.xml
+	      $(DOCBOOK2TEXI) filter-info.xml
 	      perl -ne 's/conceptindex/cindex/;s/optionindex/findex/;print;' \
 		<exim_filtering.texi | ./Tidytxt >filter.texinfo
 	      /bin/rm -rf exim_filtering.texi
@@ -198,7 +201,7 @@ spec.txt: spec-txt.html Tidytxt
 # work.
 
 spec.info:    spec-info.xml
-	      docbook2texi spec-info.xml
+	      $(DOCBOOK2TEXI) spec-info.xml
 	      ./TidyInfo <the_exim_mta.texi >spec.texinfo
 	      /bin/rm -rf the_exim_mta.texi
 	      makeinfo -o spec.info --no-split spec.texinfo
@@ -283,7 +286,7 @@ test.txt:     test-txt.xml Tidytxt MySty
 # work.
 
 test.info:    test-info.xml
-	      docbook2texi test-info.xml
+	      $(DOCBOOK2TEXI) test-info.xml
 	      ./TidyInfo <short_title.texi >test.texinfo
 	      /bin/rm -rf short_title.texi
 	      makeinfo -o test.info test.texinfo
